### ISSUE TITLE
When benchmarks fails, then workflow should fail

### ISSUE THREAD

**Issue by dgafka** [ecotoneframework/ecotone-dev#224]
Currently when benchmark fails, then workflow is considered green: https://github.com/ecotoneframework/ecotone-dev/actions/runs/6421495754/job/17435982502?pr=221

**Comment by dgafka** [comment]
@jlabedo is this something you can take a look at?

**Comment by marmichalski** [comment]
```
vendor/bin/phpbench run --retry-threshold=5 --report=github-report $PHPBENCH_REF_OPTION | bin/phpbench-to-md.sh > benchmark.md
```

This will never fail without `pipefail` option when phpbench exits with non-zero code.

Consider the following examples
| look ma im pipin                           | pipefail | exit |
|--------------------------------------|---------|------|
| `bash +o pipefail -c "false \| true"` | ❌       | 0     |
| `bash -o pipefail -c "false \| true"` | ✅        | 1     |

You would probably also need to run bash with `-e` - not sure how github executes multiple commands in the multiline run. It's easy to check though.

Anyway, this should _just work_ (tm), if you explicitly specify shell in the job, that is: `shell: bash`

**Comment by dgafka** [comment]
Fixed with #225
